src/application/cli/yargs.ts

```diff
@@ -0,0 +1,40 @@
+import yargs, { ArgumentsCamelCase, Argv } from 'yargs';
+import { hideBin } from 'yargs/helpers';
+import { CreateEntryCommand } from './create-entry-command';
+
+export interface CreateEntryArgs {
+  rulesetRepo: string;
+}
+
+export type ApplicationArgs = CreateEntryArgs;
+
+export function createParser(
+  createEntryCommand: CreateEntryCommand
+): Argv<ApplicationArgs> {
+  return yargs(hideBin(process.argv))
+    .parserConfiguration({
+      'camel-case-expansion': true,
+    })
+    .scriptName('publish-to-bcr')
+    .usage('$0 <cmd> [args]')
+    .command(
+      'create-entry',
+      'Create a new module version entry for the BCR',
+      (yargs) => {
+        yargs.option('ruleset-repo', {
+          describe:
+            'Ruleset repository containing .bcr folder with templates. Can be a remote url, e.g. `git@github.com:org/repo.git` or local path, e.g. `./path/to/repo`.',
```

Comment:
Unlike the cloud function which needs to clone the ruleset repo, if the CLI is run in an action then the repo will already have been checked out, so support a local path. This could also be useful if people want to run the CLI as a local tool to form their BCR entry.